From 66cfcf6a84e2b87db6eafeba1b950792c6830817 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Tue, 18 Feb 2014 09:37:49 -0500
Subject: [PATCH 40/74] Don't emit "Booting ..." message.

UI team still hates this stuff, so we're disabling it for RHEL 7.

Resolves: rhbz#1023142

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 grub-core/normal/menu.c       | 2 ++
 grub-core/normal/menu_entry.c | 3 ---
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/grub-core/normal/menu.c b/grub-core/normal/menu.c
index 719e2fb..89ced26 100644
--- a/grub-core/normal/menu.c
+++ b/grub-core/normal/menu.c
@@ -810,9 +810,11 @@ static void
 notify_booting (grub_menu_entry_t entry,
 		void *userdata __attribute__((unused)))
 {
+#if 0
   grub_printf ("  ");
   grub_printf_ (N_("Booting `%s'"), entry->title);
   grub_printf ("\n\n");
+#endif
 }
 
 /* Callback invoked when a default menu entry executed because of a timeout
diff --git a/grub-core/normal/menu_entry.c b/grub-core/normal/menu_entry.c
index 62c7e16..cc41fa1 100644
--- a/grub-core/normal/menu_entry.c
+++ b/grub-core/normal/menu_entry.c
@@ -1156,9 +1156,6 @@ run (struct screen *screen)
   char *dummy[1] = { NULL };
 
   grub_cls ();
-  grub_printf ("  ");
-  grub_printf_ (N_("Booting a command list"));
-  grub_printf ("\n\n");
 
   errs_before = grub_err_printed_errors;
 
-- 
2.4.3

